### PR TITLE
Updates geospatial data best practices doc based on feedback

### PR DESCRIPTION
**Pull request by jswiesner**
Includes the following changes:
-  Adds a section to clarify that VIP exclusively uses the WGS 84 datum
- Clarifies that requirements for precinct shapes apply to the "Precinct" element, not a literal precinct. This should help clarify that these requirements apply to precincts as well as precinct splits, whichever the Precinct element represents.
- Adds detail around the use case of multiple polygons to define a single Precinct boundary.

### CODE REVIEW

File: docs/data_best_practices.rst [hunk 1]
```diff
@@ -203,6 +203,16 @@ the boundary of voter precincts. In places where voter precinct shapes
...
 lightweight to integrate with existing GIS tooling.
 
+Geodetic Datum
+~~~~~~~~~~~~~~
+
+VIP exclusively uses the 84 revision of the World Geodetic System (WGS
+84) as the geodetic reference system by which geospatial coordinates
+are defined. This applies to geospatial coordinates provided within the
+VIP feed itself (e.g. PollingLocation.LatLng) as well as coordinates
+provides in an external geospatial file.
```
Comment (afsmythe) [inline docs/data_best_practices.rst:213]: typo: provides > provided

**Review by afsmythe — CHANGES_REQUESTED** [review]
Minor typo otherwise looks good, thanks!

File: docs/data_best_practices.rst (hunk 1 above)
Comment (jswiesner) [reply docs/data_best_practices.rst]: Good catch, fixed.

**Review by afsmythe — APPROVED** [review]
LG!

**Pull request merged**